.rubocop.yml

```diff
@@ -2,6 +2,12 @@
 inherit_from:
   /usr/share/YaST2/data/devtools/data/rubocop_yast_style.yml
 
+AllCops:
+  TargetRubyVersion: 2.3
```

Comment:
You can switch to the newer Rubocop by changing the config file above to `rubocop-0.71.0_yast_style.yml`. That already uses `TargetRubyVersion: 2.6`...

Reply:
Yes, you are right. But I would need to do many changes and I rather prefer to not introduce more changes. But I will open a new PR later addressing it.